packages/amplication-client/src/Components/EntitySelectField.tsx

```diff
@@ -11,27 +11,37 @@ type TEntities = {
   ];
 };
 
+type RelatedEntityOption = {
+  value: string;
+  label: string;
+};
+
 type Props = Omit<SelectFieldProps, "options"> & {
   resourceId: string;
 };
 
 const EntitySelectField = ({ resourceId, ...props }: Props) => {
+  const [relatedEntityListOptions, setRelatedEntityListOptions] =
+    React.useState<RelatedEntityOption[]>([]);
   const { data: entityList } = useQuery<TEntities>(GET_ENTITIES, {
     variables: {
       resourceId: resourceId,
     },
+    fetchPolicy: "no-cache",
```

Comment:
instead of no-cache we should use refetchQueries option on the mutation in the import page (packages/amplication-client/src/Entity/ImportPrismaSchema/EntitiesImport.tsx)

Reply:
refetchQueries is an option to refetch different query under specific mutation... why do you want to make two requests of the same query when you can do only one. other than that, we are managing our global/local state under React and not under Apollo. so, no-cache is important here as the component render it should get "fresh" data from server.